src/riverqueue/driver/riversqlalchemy/sql_alchemy_driver.py

```diff
@@ -208,11 +148,19 @@ def _build_insert_many_params(
         )
         insert_many_params.state.append(cast(models.RiverJobState, insert_params.state))
         insert_many_params.tags.append(",".join(insert_params.tags))
+        insert_many_params.unique_key.append(insert_params.unique_key or None)
+
+        if insert_params.unique_states:
+            one_byte = insert_params.unique_states[0]
+            bit_string = format(one_byte, "08b")
+            insert_many_params.unique_states.append(bit_string)
+        else:
+            insert_many_params.unique_states.append(None)
```

Comment:
@brandur the remaining lint issues are the result of the `unique_key` and `unique_states` params being typed incorrectly. Sqlc generates these as arrays whose elements are not nullable (no `None` allowed).

I've searched and scoured and haven't found any way to override the sqlc python plugin to achieve this. Any ideas?

Reply:
Thanks, used this approach to fix the type issue :pray: